packages/orchestrator-ui-components/src/components/WfoSubscription/WfoInUseByRelations.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+import { useTranslations } from 'next-intl';
+import Link from 'next/link';
+
+import {
+    WfoError,
+    WfoFirstPartUUID,
+    WfoKeyValueTable,
+    WfoLoading,
+} from '@/components';
+import type { WfoKeyValueTableDataType } from '@/components';
+import { useWithOrchestratorTheme } from '@/hooks';
+import { useGetInUseByRelationDetailsQuery } from '@/rtk';
+import { InUseByRelation, InUseByRelationDetail } from '@/types';
+
+import { getStyles } from './styles';
+
+interface WfoInUseByRelationsProps {
+    inUseByRelations: InUseByRelation[];
+}
+
+export const WfoInUseByRelations = ({
+    inUseByRelations,
+}: WfoInUseByRelationsProps) => {
+    const t = useTranslations('subscriptions.detail');
+    const { inUseByRelationDetailsStyle } = useWithOrchestratorTheme(getStyles);
+    const subscriptionIds = inUseByRelations
+        .map((relation) => relation.subscription_id)
```

Comment:
In the past I used lodash (and similar libraries) to get values from objects and avoid having to endlessly write mapping function.